app/controllers/schools/meters_controller.rb

```diff
@@ -15,15 +15,23 @@ def index
     end
 
     def show
+      @n3rgy_status = MeterManagement.new(@meter).check_n3rgy_status
```

Comment:
There is potential for the status call to raise an error if the API returns something unexpected - could handle in the API, or handle here? (or in the meter management)